test/presenters/taxon_presenter_test.rb

```diff
@@ -75,19 +75,6 @@
     end
   end
 
-  # describe 'supergroup_sections' do
-  #   it 'returns a list of supergroup details' do
-  #     taxon = mock
-  #     taxon.stubs(:section_content).returns([])
-  #     taxon.stubs(:base_path)
-  #     taxon_presenter = TaxonPresenter.new(taxon)
-
-  #     taxon_presenter.sections.each do |section|
-  #       assert_equal(%i(show_section title documents see_more_link), section.keys)
-  #     end
-  #   end
-  # end
-
```

Comment:
For neatness, I'd move this deletion to the first commit, where the test is first commented out.

Reply:
Ok sure